doc/mgr/restful.rst

```diff
@@ -110,6 +110,14 @@ You can use ``curl`` in order to test your user with the API. Here is an example
 
 In the case above, we are using ``GET`` to fetch information from the ``server`` endpoint.
 
+To simplify the operation, you can ignore API user authentication with the following command::
+
+  $ ceph restful disable-http-auth
```

Comment:
We are serving restful over https. You can just remove the `http` bit from the commands and have `disable-auth` and `enable-auth` commands.